spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -402,7 +402,7 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateDoubles(), DataTypes.FloatType)
   }
 
-  ignore("cast DoubleType to DecimalType(10,2)") {
+  test("cast DoubleType to DecimalType(10,2)") {
     // Comet should have failed with [NUMERIC_VALUE_OUT_OF_RANGE]
```

Comment:
This comment can be removed now.

```suggestion
    // Comet should have failed with [NUMERIC_VALUE_OUT_OF_RANGE]
```